src/applications/coronavirus-vaccination/components/Introduction.jsx

```diff
@@ -112,21 +119,61 @@ function Introduction({
           content={
             <>
               <p>
-                You don’t need to sign up to get a vaccine. And you can change
-                your mind about getting a vaccine at any time. We’ll use the
-                information you provide to understand your interest and keep you
-                informed.
+                <strong>What you should know about signing up</strong>
               </p>
               <p>
-                <a href="/health-care/covid-19-vaccine/#who-will-get-a-covid-19-vaccin">
-                  Learn who will get a COVID-19 vaccine first based on CDC
-                  guidelines
+                We continue to contact Veterans as they become eligible to get a
+                COVID-19 vaccine. We base eligibility on VA and{' '}
+                <a
+                  href="https://www.cdc.gov/vaccines/covid-19/phased-implementation.html"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  aria-label="CDC COVID-19 risk criteria phased implemention (Open in a new window)"
+                >
+                  CDC COVID-19 risk criteria
                 </a>
+                . Within each risk group, we may first contact Veterans who sign
+                up here and tell us that they plan to get a vaccine. But we’ll
+                still contact every eligible Veteran in each risk group to ask
+                if they want to get a vaccine. You don’t need to sign up here,
+                call, or come to a VA facility to request or reserve a vaccine.
+              </p>
+              <p>
+                By sharing your plans for getting a vaccine, you help us better
+                plan our efforts. This helps us do the most good with our
+                limited vaccine supply.
               </p>
+              <span>
+                <a
+                  href="/health-care/covid-19-vaccine/#who-will-get-a-covid-19-vaccin"
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  aria-label="Learn who will get a COVID-19 vaccine first based on CDC
+                  guidelines (Open in a new window)"
+                >
+                  Learn who can get a COVID-19 vaccine now based on CDC
+                  guidelines
+                </a>
+              </span>
             </>
           }
         />
       </DowntimeNotification>
+      <br />
+      <span>
```

Comment:
Instead of the `br` we could switch out the `span` with a `div` and add the class `vads-u-margin-top--1` to it